loky/backend/utils.py

```diff
@@ -84,3 +94,221 @@ def _recursive_terminate(pid):
             # level function raise a warning and retry to kill the process.
             if e.errno != errno.ESRCH:
                 raise
+
+
+##############################################################################
+# The following code is derived from code by Intel developper @anton-malakhov
```

Comment:
derived or inspired?
In both cases I think it useful to recall the original copyright and license:

"Copyright (c) 2017, Intel Corporation published under the BSD 3-Clause license"